python/pyspark/sql/streaming.py

```diff
@@ -1501,6 +1501,16 @@ def toTable(self, tableName, format=None, outputMode=None, partitionBy=None, que
         A new table will be created if the table not exists. The returned
         :class:`StreamingQuery` object can be used to interact with the stream.
 
+        Note, if the table does not exist, both V1 and V2 tables will respect the
```

Comment:
I think we can move these docs under `Notes` before.
Also, maybe `partitioningColumns` -> `partitioning columns`?

Reply:
Thanks, done in c158775